data-service/src/test/java/uk/gov/laa/ccms/data/mapper/LookupMapperImplTest.java

```diff
@@ -316,6 +318,38 @@ void toRelationshipToCaseLookupDetailTest() {
 
 
 
+
+
+    @Test
+    void toAwardTypeLookupValueDetailTest() {
+        AwardTypeLookupValue value = createAwardTypeLookupValue("1");
+
+        AwardTypeLookupValueDetail expectedDetail = createAwardTypeLookupValueDetail(value);
+        AwardTypeLookupValueDetail actualDetail =
+            mapper.toAwardTypeLookupValueDetail(value);
+
+        assertEquals(expectedDetail, actualDetail);
+    }
+
+    @Test
+    void toAwardTypeLookupDetailTest() {
```

Comment:
does this miss coverage on the Page mapping? Other lookups do a page match which uses the list mapping.